provision-lms.sh

```diff
@@ -28,6 +28,10 @@ docker-compose exec lms bash -c 'source /edx/app/edxapp/edxapp_env && cd /edx/ap
 docker-compose exec lms bash -c 'source /edx/app/edxapp/edxapp_env && python /edx/app/edxapp/edx-platform/manage.py lms --settings=devstack_docker manage_user edx edx@example.com --superuser --staff'
 docker-compose exec lms bash -c 'source /edx/app/edxapp/edxapp_env && echo "from django.contrib.auth import get_user_model; User = get_user_model(); user = User.objects.get(username=\"edx\"); user.set_password(\"edx\"); user.save()" | python /edx/app/edxapp/edx-platform/manage.py lms shell  --settings=devstack_docker'
 
+# Create a superuser for amc
+docker-compose exec lms bash -c 'source /edx/app/edxapp/edxapp_env && python /edx/app/edxapp/edx-platform/manage.py lms --settings=devstack_docker manage_user amc amc@example.com --superuser --staff'
+docker-compose exec lms bash -c 'source /edx/app/edxapp/edxapp_env && echo "from django.contrib.auth import get_user_model; User = get_user_model(); user = User.objects.get(username=\"amc\"); user.set_password(\"amc\"); user.save()" | python /edx/app/edxapp/edx-platform/manage.py lms shell  --settings=devstack_docker'
```

Comment:
A similar functionality exists in 

https://github.com/appsembler/devstack/blob/ed2fb0d18f81ecadf7229c0708a84a1f4885211f/tahoe.mk#L45

Reply:
Not really, this management command actually creates a superuser in AMC only

Reply:
@ahmedaljazzar, true. Then you can use the `edx:edx` user with the email `edx@example.com`. 

I don't see why we need to add another `docker-compose exec lms` knowing that it's pretty heavy command to run and would slowdown the provision process even more.

Reply:
I see your point. If you can take a look at this change I'm trying to do [here](https://github.com/appsembler/amc/pull/197/files#diff-ef933aedf379543d20226ab9ad8416b0R18). Basically, I want this user to have an oath token so I can use it in the API calls, with AMC access permission to perform off boarding on edX side.